src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/admin/controllers/AdminController.java

```diff
@@ -142,16 +150,21 @@ GetUnreadRegistrationApplicationCountResponse registrationApplicationCount(
 
     aclService.canPerson(person)
         .performTheAction(RolePermissionInstanceTypes.INSTANCE_REMOVE_ADMIN)
+
         .orElseThrow(() -> new ResponseStatusException(HttpStatus.UNAUTHORIZED, "not_an_admin"));
 
-    return GetUnreadRegistrationApplicationCountResponse.builder().registration_applications(
-        (int) personRegistrationApplicationRepository.countByApplicationStatus(
-            PersonRegistrationApplicationStatus.pending)).build();
+    return GetUnreadRegistrationApplicationCountResponse.builder()
+        .registration_applications(
```

Comment:
(Doesn't need to be in this PR, but...) can we rename `registration_applications` to use camelCase?